### DISCUSSION TITLE
Assignee from CODEOWNERS not working for GitLab CI Components

### DISCUSSION THREAD

**Discussion by OrbisK** [renovatebot/renovate#29411] [Request Help] [answered]
### What would you like help with?

I think I found a bug

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab 17, Renovate latest

### Please tell us more about your question or problem

```CODEOWNERS
# All files
* @me @someoneelse

[GitLab CI] @me
*.gitlab-ci.yml
.gitlab-ci.yml
.gitlab/
```

```json
{
  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
  "extends": [
    "config:recommended",
    ":preserveSemverRanges",
    ":dependencyDashboard",
    ":rebaseStalePrs",
    "group:recommended",
    "mergeConfidence:all-badges"
  ],
  "osvVulnerabilityAlerts": true,
  "assigneesFromCodeOwners": true
}


```

Updates for Gitlab CI Components are not assigned to me :(

### Logs (if relevant)

<details><summary>Logs</summary>

No warning from codeowners: https://github.com/renovatebot/renovate/blob/f60b3e24acddab663e465345d45bc01b55bd1c5c/lib/workers/repository/update/pr/code-owners.ts#L165

I can add debug logs if needed.
</details>

**Comment by secustor** [comment, marked as answer]
Gitlab default owners are currently not supported https://github.com/renovatebot/renovate/issues/29202

**Reply by kastl-ars** [reply]
I take it that this is still not implemented (according to the issue you linked to).
